src/dbg/memory.cpp

```diff
@@ -258,6 +258,17 @@ static void ProcessFileSections(std::vector<MEMPAGE> & pageVector, std::vector<S
             for(const auto & page : newPages)
                 totalSize += page.mbi.RegionSize;
 
+            if(totalSize < pageSize)
+            {
+                // On Windows 11 24H2+, modules that support hotpatching contain an extra page mapped into their image region
+                MEMPAGE imageExtensionPage = {};
+                imageExtensionPage.mbi.BaseAddress = (PVOID)(pageBase + totalSize);
+                imageExtensionPage.mbi.RegionSize = pageSize - totalSize;
+                sprintf_s(imageExtensionPage.info, "ImageExtension");

```

Comment:
"ImageExtension" is the name that shows up in kernel functions which deal with the extra page, hence why I named it this. If the format should be different, let's change it.